src/components/MDXComponents/__tests__/MDXCode.test.tsx

```diff
@@ -44,6 +44,15 @@ describe('MDXCode', () => {
     expect(titleElement.className).toContain('pre-title');
   });
 
+  it('should reset the code-block css counter', async () => {
+    // line numbers have been moved to css, so here we are looking for the class that has "counter-reset: code-block;"
+    // line number css can be found in `/src/styles/code.scss`
+    const { container } = render(
+      <MDXCode testId={testId} codeString={codeString}></MDXCode>
+    );
+    expect(container.querySelector('.code-block')).toBeInTheDocument();
```

Comment:
`getComputedStyle` was not returning the styles for the elements so I was unable to test actual css values on these elements.  Instead I am just validating that the needed classes are present on the elements that would produce the line numbers.